scipy/optimize/_lsq/lsq_linear.py

```diff
@@ -32,7 +32,8 @@ def prepare_bounds(bounds, n):
 
 
 def lsq_linear(A, b, bounds=(-np.inf, np.inf), method='trf', tol=1e-10,
-               lsq_solver=None, lsmr_tol=None, max_iter=None, verbose=0):
+               lsq_solver=None, lsmr_tol=None, max_iter=None,
+               verbose=0, lsmr_max_iter=None,):
```

Comment:
We ask to define new arguments as keyword only for easier maintenance + explicitness.
```suggestion
               verbose=0, *, lsmr_max_iter=None,):
```